src/app/browser/qgsinbuiltdataitemproviders.cpp

```diff
@@ -2037,7 +2037,7 @@ bool QgsDatabaseItemGuiProvider::handleDropUri( QgsDataItem *item, const QgsMime
     }
   };
 
-  auto onFailure = [connectionItemPointer]( Qgis::VectorExportResult, const QString & ) {
```

Comment:
move operator seems to be defined only in [6.6](https://doc.qt.io/qt-6/qpointer.html#operator-eq). Wouldn't it cause some issue for version below ? Why do we need to explicit move here ?

Reply:
Implicit copy operator should kick in for those earlier versions. Explicit move is beneficial because the variable isn't used after this line.

Reply:
OK thanks for the details.